src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -186,6 +189,20 @@ public void setBook(Book target, Book editedBook) {
         return catalog.getOverdueBooks();
     }
 
+    /**
+     * Returns a list of overdue books in the catalog.
+     *
+     * @return an <code>ObservableList</code> of overdue books.
+     */
+    @Override
+    public ObservableList<Borrower> getOverdueBooksBorrowers() {
+        List<Borrower> borrowers = catalog.getOverdueBooksBorrowersId()
+                .stream()
+                .map(borrowerId -> borrowerRecords.getBorrowerFromId(borrowerId))
+                .collect(Collectors.toList());
+        return FXCollections.observableArrayList(borrowers);
+    }
```

Comment:
remove this method as per previous commits

Reply:
Also take note to remove this method from the `Model` interface and from `AddCommandTest`

Reply:
Will appreciate your help to help me remove the `getOverdueBooksBorrowersId()` method from `Catalog.java` as well! Thanks so much for your help :)